4-spring-web-app/2-Authorization-I/call-graph/pom.xml

```diff
@@ -15,7 +15,7 @@
   <name>ms-identity-spring-boot-webapp</name>  
   <description>Microsoft identity platform Spring Boot sample</description>  
   <properties> 
-    <java.version>11</java.version>  
+    <java.version>8</java.version>
```

Comment:
Why downgrade this? Generally our samples should be on the "latest and greatest" and our libraries should support the lowest.